<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>friendDetailDialog</class>
 <widget class="QDialog" name="friendDetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="nicknameLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>好友没有设置昵称</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>390</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删除好友</string>
   </property>
  </widget>
  <widget class="QLabel" name="avatarLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>好友头像</string>
   </property>
  </widget>
  <widget class="QLabel" name="signatureLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>581</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>好友没有设置个性签名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>390</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>活动爱好</string>
   </property>
  </widget>
  <widget class="QLabel" name="hobbyLabel1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>爱好1: 暂未设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="hobbyLabel2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>爱好2：暂未设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="IDLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>好友ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_frame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>751</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255,0.95);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>1</y>
     <width>901</width>
     <height>601</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_hobbyframe">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>361</y>
     <width>351</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>label_frame</zorder>
  <zorder>label_hobbyframe</zorder>
  <zorder>nicknameLabel</zorder>
  <zorder>deleteButton</zorder>
  <zorder>avatarLabel</zorder>
  <zorder>signatureLabel</zorder>
  <zorder>label</zorder>
  <zorder>hobbyLabel1</zorder>
  <zorder>hobbyLabel2</zorder>
  <zorder>IDLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
